edx_proctoring/api.py

```diff
@@ -1309,6 +1309,11 @@ def _resolve_prerequisite_links(exam, prerequisites):
         'short_description': _('Failed Proctoring'),
         'suggested_icon': 'fa-exclamation-triangle',
         'in_completed_state': True
+    },
+    ProctoredExamStudentAttemptStatus.expired: {
+        'short_description': _('Proctored Option Not Available'),
```

Comment:
Actually, I think it would be better to display no message in the summary at all. See comment below.